inception-recommendation-api/src/main/java/de/tudarmstadt/ukp/inception/recommendation/api/evaluation/EvaluationResult.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2018
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.recommendation.api.evaluation;
+
+public class EvaluationResult
+{
+    private double defaultScore;
+    private int trainingSetSize;
+    private int testSetSize;
+
+    /**
+     * Get default score of the recommender evaluation.
+     * @return evaluation score 
+     */
+    public double getDefaultScore()
+    {
+        return defaultScore;
+    }
+
+    /**
+     * Set the default score of the recommender evaluation.
+     * @param aDefaultScore
```

Comment:
Either describe the parameter or do not declare it. I'd personally tend to not declaring it since the parameter description would be very redundant with the method description... but it's your call.